collections/Laud_Misc/MS_Laud_Misc_97.xml

```diff
@@ -37,46 +36,256 @@
                      <author key="person_95155322">Origen</author>
                      <title key="work_3395">Homilies on Joshua</title>
-                     <note>(tr. <persName key="person_77679446" role="trl"> Rufinus </persName>)</note>
+                     <note>(tr. <persName key="person_77679446" role="trl"> Rufinus
+                        </persName>)</note>
+                     <note>Ed. W. A. Baehrens, GCS 30 (1921), pp. 286-463; the quires misbound after fol. 29, as noted below.</note><!-- https://archive.org/details/origeneswerke07orig/page/352/mode/2up -->
+                    <msItem>
+                       <note>Fols. 1-29: quires 1-4 (a-d)</note>
+                       <msItem>
+                        <locus>(fol. 1r-v)</locus>
+                          <rubric>Incipit prologus Hieronimi super librum Iesu Naue ad Chromacium episcopum</rubric>
+                          <incipit>In diuinis uoluminibus refertur</incipit>
+                          <explicit>expetis ab indoctis</explicit>
+                          <note>Prologue; the lower half of fol. 1v is filled with the rubric for the following item.</note>
+                     </msItem>
+                     <msItem>
+                        <locus>(fol. 1v-6r)</locus>
+                        <rubric>Incipit omelia prima tractatus super Iesu Nave</rubric>
+                        <incipit><locus>(fol. 2r)</locus> Donauit Deus nomen quod est super</incipit>
+                        <note> Homilia I</note>
+                     </msItem>
+                     <msItem>
+                        <locus>(fols. 6r-7r)</locus><note> Homilia II</note>
+                     </msItem>
+                     <msItem>
+                        <locus>(fols. 7r-11r)</locus><note> Homilia III</note>
+                     </msItem>
+                     <msItem>
+                        <locus>(fols. 11r-14r)</locus><note> Homilia IV</note>
+                     </msItem>
+                     <msItem>
+                        <locus>(fols. 14r-17v)</locus><note> Homilia V</note>
+                     </msItem>
+                     <msItem>
+                        <locus>(fols. 17v-19r)</locus><note> Homilia VI</note>
+                     </msItem>
+                     <msItem>
+                        <locus>(fols. 19r-23v)</locus><note> Homilia VII</note>
+                     </msItem>
+                     <msItem>
+                        <locus>(fols. 23v-26(b)v)</locus><note> Homilia VIII</note>
+                     </msItem>
+                     <msItem>
+                        <locus>(fols. 26(b)v-29v)</locus><note> Homilia IX.1-7</note>
+                        <explicit>Alii uero sunt qui et <note>(ed. Baehrens 352.5)</note></explicit>
+                     </msItem></msItem>
+                     
+                     
+                   <msItem>
+                      <note>Fols. 30-35: quire 9</note>
```

Comment:
Is there any way to make these headings more prominent? I am almost tempted to suggest dealing with these misplaced quires using `<msPart>`, but it would probably add too much complexity.